libethereum/Client.cpp

```diff
@@ -40,7 +40,7 @@ VersionChecker::VersionChecker(string const& _dbPath):
 {
 	auto protocolContents = contents(m_path + "/protocol");
 	auto databaseContents = contents(m_path + "/database");
-	m_ok = RLP(protocolContents).toInt<unsigned>(RLP::LaisezFaire) == eth::c_protocolVersion && RLP(databaseContents).toInt<unsigned>(RLP::LaisezFaire) == c_databaseVersion;
```

Comment:
for good practice, should be explicitly `eth::c_protocolVersion` since we now have `p2p::c_protocolVersion`.